.dockerignore

```diff
@@ -14,5 +14,5 @@ var/
 vendor/
 
 # env files are copied into the final layer
-.env
```

Comment:
I thought we needed the .env file, though I could be wrong.  The docker-compose file adds the env file args to the environment, so maybe it's not required.

Reply:
Notes from @disfated in Discord, with some things bolded by me:

* Your current setup relies on docker-compose to build and run a docker container. Docker compose is just a script that under the hood submits normal docker commands for you. So when you docker compose up it effectively performs docker build producing an image, which is actually broken if you try to run it independently, the reason it works is that docker compose subsequently runs docker run ... --env-file .env, passing those envvars into the container. If you were to manually run the image (docker run lute_image), the application wouldn't even know where to find the database. So this is not a viable image, because **you always have to make docker images as self-sufficient and self-reliant as possible!** Then I noticed that if I include .env file into the container, PHP actually reads it and luckily the application works. While this isn't a perfect solution, but it's just what I have to do to make a functional image, **which can be run with zero or minimal config** just about anywhere where docker runs.

* By the way, all those envvars in the .env file don't really belong there because they are actually user preferences (**ideally should be adjustable from within the application's UI**) or some internal constants (like DB_FILENAME) that should reside in PHP or some kind of config, like /lute/data/config.format.

Agreed on the "user settings" that should be in the UI, the current .env config file was just to get things working quickly. :-)

Reply:
Side note: the .env file is required for the Symfony framework, afaict.  Even though it's loaded into the environment with each request, I can't say that it is removable.  Regardless, if all of the user setting/pref stuff (i.e. the backup settings) are moved into the UI, the user won't need to care about the .env file.